maven/api-maven/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/coordinate/MavenCoordinateImpl.java

```diff
@@ -36,11 +36,11 @@ class MavenCoordinateImpl extends MavenGABaseImpl implements MavenCoordinate {
      * required. If no {@link PackagingType} is specified, default will be to {@link PackagingType#JAR}. If no {@link ScopeType}
      * is specified, default will be {@link ScopeType#COMPILE}.
      *
-     * @param groupId
-     * @param artifactId
-     * @param version
-     * @param packaging
-     * @param classifier
+     * @param groupId     The group ID of the Maven coordinate.
```

Comment:
Why the extra space here?

Reply:
It looks correct to me, so the descriptions are lined up for better readability in IDE